hardware/arduino/avr/cores/arduino/Stream.h

```diff
@@ -1,6 +1,5 @@
 /*
-  Stream.h - base class for character-based streams.
-  Copyright (c) 2010 David A. Mellis.  All right reserved.
```

Comment:
Nothing is left of the original file apart from the name `Stream`, so this copyright notice has moved to `IStream.h`